### PR TITLE
Use react-paypal-js for one time PayPal payments

### PR DESCRIPTION
**Pull request by HanaXu**
## Description
Use the react-paypal-js package to handle PayPal one time payments

## How to test
1. Point Cloud Manager to the dev environment
2. Go to `account/billing/make-payment`
3. Click the yellow PayPal button and go through the sandbox flow (login using sandbox credentials in password manager)
   - The difference between this branch and prod is that after making a payment in the sandbox flow, the payment goes through and the drawer closes; there is no additional confirmation modal

### CODE REVIEW

File: packages/manager/src/features/Billing/BillingPanels/BillingSummary/PaymentDrawer/PayPalButton.tsx [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+  const [
+    payPalInitializationError,
+    setPayPalInitializationError,
+  ] = React.useState<boolean>(false);
+
+  const [
+    payPalCheckoutInstance,
+    setPayPalCheckoutInstance,
+  ] = React.useState<any>();
```
Comment (codyfinn) [inline packages/manager/src/features/Billing/BillingPanels/BillingSummary/PaymentDrawer/PayPalButton.tsx:78]: Could we change this `any` to a `unknown` and an if statement?

File: packages/manager/src/features/Billing/BillingPanels/BillingSummary/PaymentDrawer/PaymentDrawer.tsx [hunk 2]
```diff
@@ -350,144 +332,82 @@ export const PaymentDrawer: React.FC<Props> = (props) => {
...
-                        />
+                        ? getTPPIcon(type)
+                        : getCreditCardIcon(card_type);
+                    return <Icon />;
+                  };
+
+                  const renderVariant = () => {
+                    return is_default ? (
+                      <Grid item className={classes.chip} xs={3} md={2}>
+                        <Chip label="DEFAULT" component="span" />
```
Comment (codyfinn) [inline packages/manager/src/features/Billing/BillingPanels/BillingSummary/PaymentDrawer/PaymentDrawer.tsx:380]: Could this be refactored into a generic PaymentMethod component that depending upon its props returns either a CreditCardPaymentMethod component or a ThirdPartyPaymentMethod. Might clean up some of the ternary logic

Comment (bnussman) [reply packages/manager/src/features/Billing/BillingPanels/BillingSummary/PaymentDrawer/PaymentDrawer.tsx:380]: @HanaXu and I can work on a follow up PR to clean stuff like this up

**Review by bnussman — APPROVED** [review]
Functionality is great! 

One unrelated piece that I think would be nice would be to add would be adding the code below to the **Payment Amount** `TextField` in `PaymentDrawer.tsx`
```ts
InputProps={{
  startAdornment: (
    <InputAdornment position="end">$</InputAdornment>
  ),
}}
```
Screen Shot 2021-11-04 at 6 06 01 PM

**Review by tiffwong — APPROVED** [review]
Flow worked as expected for me, great job! 🎉

**Pull request merged**